common.sh

```diff
@@ -39,6 +39,16 @@ source $CONFIG
 export OPENSHIFT_RELEASE_IMAGE="${OPENSHIFT_RELEASE_IMAGE:-registry.svc.ci.openshift.org/ocp/release:4.2}"
 export OPENSHIFT_INSTALL_PATH="$GOPATH/src/github.com/openshift/installer"
 
+# Switch Container Images to upstream, Installer defaults these to the openshift version
+if [ "$UPSTREAM_IRONIC" == "True" ] ; then
```

Comment:
This variable is undefined by default, perhaps we should define it above with a default of "false", then check here for != "false" so we don't have to worry about case-sensitive "True"?